### PR TITLE
Update Civil and Environmental Engineering Major (CEE)

### PR DESCRIPTION
**Pull request by richardhuangz**

### CODE REVIEW

File: tigerpath/majors_and_certificates/majors/CEE.json [hunk 1]
```diff
@@ -7,11 +7,11 @@
   "min_needed": 1,
   "urls": [
     "https://ua.princeton.edu/academic-units/department-civil-and-environmental-engineering",
-    "http://www.princeton.edu/cee/forms/Yellow_Book_for_Class_of_2020_Mar.2017_3.7.pdf"
+    "https://cee.princeton.edu/undergraduate/paths-graduation"
   ],
   "contacts": [
     {
-      "type": "Departmental Representative",
```
Comment (richardhuangz) [inline tigerpath/majors_and_certificates/majors/CEE.json:14]: https://ua.princeton.edu/academic-units/department-civil-and-environmental-engineering

File: tigerpath/majors_and_certificates/majors/CEE.json [hunk 2]
```diff
@@ -41,14 +41,23 @@
               "min_needed": null,
               "explanation": null,
               "course_list": [
-                "CEE 205: Mechanics of Solids",
-                "CEE 262A: Structures and the Urban Environment",
```
Comment (richardhuangz) [inline tigerpath/majors_and_certificates/majors/CEE.json:45]: https://ua.princeton.edu/academic-units/department-civil-and-environmental-engineering

File: tigerpath/majors_and_certificates/majors/CEE.json [hunk 3]
```diff
@@ -41,14 +41,23 @@
...
                 "CEE 365: Soil Mechanics"
               ]
             },
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "CEE 205: Mechanics of Solids",
```
Comment (richardhuangz) [inline tigerpath/majors_and_certificates/majors/CEE.json:57]: https://ua.princeton.edu/academic-units/department-civil-and-environmental-engineering

"CEE 205 Mechanics of Solids or MAE 223 Modern Solid Mechanics"

This same change occurs multiple times

File: tigerpath/majors_and_certificates/majors/CEE.json [hunk 4]
```diff
@@ -336,8 +353,18 @@
...
+                "MAE 223 Modern Solid Mechanics"
+              ]
+            },
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "CEE 302: Practical Models for Environmental Systems",
```
Comment (richardhuangz) [inline tigerpath/majors_and_certificates/majors/CEE.json:366]: "CEE 302 Practical Models for Environmental Systems or CEE 304 Environmental Engineering and Energy"

File: tigerpath/majors_and_certificates/majors/CEE.json [hunk 5]
```diff
@@ -482,28 +509,19 @@
...
                   ]
                 },
                 {
                   "name": null,
-                  "max_counted": 1,
-                  "min_needed": null,
-                  "explanation": null,
-                  "course_list": [
-                    "CEE 305/GEO 375: Environmental Fluid Mechanics",
-                    "CEE 311/GEO 311: Global Air Pollution"
```
Comment (richardhuangz) [inline tigerpath/majors_and_certificates/majors/CEE.json:496]: "CEE 311 (GEO 311) Global Air Pollution"

311 is required (not interchangeable with 305)

File: tigerpath/majors_and_certificates/majors/CEE.json (hunk 1 above)
Comment (churichard) [reply tigerpath/majors_and_certificates/majors/CEE.json:14]: This should be "Director of Undergraduate Studies"

File: tigerpath/majors_and_certificates/majors/CEE.json [hunk 6]
```diff
@@ -681,20 +699,30 @@
...
               ]
             },
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "CEE 205: Mechanics of Solids",
+                "MAE 223 Modern Solid Mechanics"
```
Comment (churichard) [inline tigerpath/majors_and_certificates/majors/CEE.json]: there should be a `:` after `MAE 223` (fix this in other places too)

**Review by churichard — COMMENTED** [review]
For each place you updated the course list, could you also update the explanations as well? You can probably remove the explicit course list in the explanation so we don't have to update things in two places.

For example, if an explanation says

> Engineering science requirements (eight courses):\nCEE 205 Mechanics of Solids\nCEE 262A Structures and the Urban Environment\nCEE 207 Introduction to Environmental Engineering, or CEE 302 Practical Models for Environmental Systems, or CEE 306 Hydrology\nCEE 312 Statics of Structures\nCEE 361 Matrix Structural Analysis and Introduction to Finite-Element Methods\nCEE 364 Materials in Civil Engineering\nCEE 365 Soil Mechanics\nARC 374 Computational Design or CEE 463 A Social and Multi-Dimensional Exploration of Structures

Just change it to

> Engineering science requirements (eight courses)

File: tigerpath/majors_and_certificates/majors/CEE.json (hunk 1 above)
Comment (richardhuangz) [reply tigerpath/majors_and_certificates/majors/CEE.json:14]: Fixed

File: tigerpath/majors_and_certificates/majors/CEE.json (hunk 6 above)
Comment (richardhuangz) [reply tigerpath/majors_and_certificates/majors/CEE.json]: Fixed

File: tigerpath/majors_and_certificates/majors/CEE.json [hunk 7]
```diff
@@ -383,7 +383,7 @@
           "name": "Engineering design",
           "max_counted": 1,
           "min_needed": 2,
-          "explanation": "Engineering design (two courses):\n\nCEE 471 Introduction to Water Pollution Technology\nCEE 477 Engineering Design for Sustainable Development",
+          "explanation": "Engineering design (two courses):\nCEE 471: Introduction to Water Pollution Technology,\nCEE 477: Engineering Design for Sustainable Development",
```
Comment (churichard) [inline tigerpath/majors_and_certificates/majors/CEE.json:386]: You can probably delete the courses in the explanation here.

Comment (churichard) [reply tigerpath/majors_and_certificates/majors/CEE.json:386]: Actually never mind, you can probably leave it. There seems to be a lot of these explanations with course lists in them. No need to change them all

**Review by churichard — APPROVED** [review]

**Pull request merged**